pkgs/by-name/ll/llm-ls/package.nix

```diff
@@ -19,21 +24,43 @@ rustPlatform.buildRustPackage {
     sha256 = "sha256-ICMM2kqrHFlKt2/jmE4gum1Eb32afTJkT3IRoqcjJJ8=";
   };
 
-  cargoHash = "sha256-Fat67JxTYIkxkdwGNAyTfnuLt8ofUGVJ2609sbn1frU=";
+  cargoHash = "sha256-m/w9aJZCCh1rgnHlkGQD/pUDoWn2/WRVt5X4pFx9nC4=";
+
+  cargoPatches = [
+    # https://github.com/huggingface/llm-ls/pull/102
+    (fetchpatch {
+      name = "fix-time-compilation-failure.patch";
+      url = "https://github.com/huggingface/llm-ls/commit/cb0a3da737d933614bcd011e91048f2009036d09.patch?full_index=1";
+      hash = "sha256-rgC0u3cOg15ECMOupJsu0eGb43+DXmDClqYEJ+fRWhM=";
+    })
+    # https://github.com/huggingface/llm-ls/commit/479401f3a5173f2917a888c8068f84e29b7dceed
+    (fetchpatch {
+      name = "fix-version.patch";
+      url = "https://github.com/huggingface/llm-ls/commit/479401f3a5173f2917a888c8068f84e29b7dceed.patch?full_index=1";
+      hash = "sha256-4gXasfMqlrrP8II+FiV/qGfO7a9qFkDQMiax7yEua5E=";
+    })
+  ];
 
   buildAndTestSubdir = "crates/llm-ls";
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [ openssl ];
+  buildInputs = [
+    openssl
```

Comment:
Have you checked whether this is necessary on Darwin?  Given that the package uses native-tls, it may not be.

Reply:
You're right, I have been able to build it without, just need `libiconv` on Darwin